pkgs/applications/version-management/git-town/default.nix

```diff
@@ -63,7 +64,7 @@ buildGoModule rec {
     description = "Generic, high-level git support for git-flow workflows";
     homepage = "https://www.git-town.com/";
     license = licenses.mit;
-    maintainers = with maintainers; [ allonsy blaggacao ];
+    maintainers = with maintainers; [ allonsy blaggacao gabyx];
```

Comment:
```suggestion
    maintainers = with maintainers; [ allonsy blaggacao gabyx ];
```

Reply:
Isn't there a format rule or make targer to do this?

Reply:
Formatting should be foremost locally consistent. If a list starts with a space, it should also end with one.

alejandra mostly produces weird code and isn't really recommendable. 

There is a formatting RFC that is being worked on.